configure.in

```diff
@@ -142,8 +142,8 @@ changequote([,])
 GRASS_VERSION_GIT="exported"
 # get git short hash + date of last change in GRASS headers
 # (and anything else in include)
-GRASS_HEADERS_GIT_HASH=`date`
-GRASS_HEADERS_GIT_DATE=`date`
+GRASS_HEADERS_GIT_HASH=`date -Iseconds`
```

Comment:
Would UTC for time zone make more sense, i.e. `date -u -Iseconds`? That way two devs in different locations will get the same result and when you see the time, you don't have to think which time zone it is since it is always UTC.

Reply:
That's a good point. It's devs' timezone-agnostic and more universal for users even though it can be confusing to ourselves (?).

Reply:
I agree, @wenzeslaus @HuidaeCho but why to mimic this behaviour by `git log`.

Reply:
@landam I'm not sure what you mean by the `git log` issue. We need to use `git log` to retrieve the commit time of header files, not the current time. AFAIK, `date -Iseconds` is only a fall back for `GRASS_HEADERS_GIT_DATE` in case git is not available. Maybe, I'm missing something?

Reply:
@HuidaeCho We need to modify https://github.com/landam/grass/blob/date_seconds/configure.in#L157 to produce timestamp in the same format as `date -u -Iseconds`

Reply:
@HuidaeCho @wenzeslaus Any idea how to change `$GIT log -1 --pretty=format:"%cI" -- "${SRCDIR}/include` in order to produce timestamp UTC zone (same as `date -u -Iseconds)?

Reply:
> @HuidaeCho @wenzeslaus Any idea how to change `$GIT log -1 --pretty=format:"%cI" -- "${SRCDIR}/include` in order to produce timestamp UTC zone (same as `date -u -Iseconds)?

Pure git.. I don't know. `TZ=UTC git log -1 --date=local` prints time in UTC, but not `TZ=UTC git log -1 --date=local --pretty=format:"%cI"` :-(

Reply:
@landam What about this? `date -d $(git log -1 --pretty=format:%cI) -u -Iseconds`

Reply:
done in d76035a48373df74f5195a2157cae2630687bb07